f5/bigip/ltm/__init__.py

```diff
@@ -13,12 +13,12 @@
 # limitations under the License.
 #
 from f5.bigip.ltm.monitor import Monitor
-from f5.bigip.ltm.nat import NATCollection
-from f5.bigip.ltm.policy import PolicyCollection
-from f5.bigip.ltm.pool import PoolCollection
-from f5.bigip.ltm.rule import RuleCollection
-from f5.bigip.ltm.snat import SNATCollection
-from f5.bigip.ltm.virtual import VirtualCollection
+from f5.bigip.ltm.nat import NATs
+from f5.bigip.ltm.policy import Policys
```

Comment:
As long as everyone is fine with Policys, I am fine. Just bringing attention to it.